lib/paranoia.rb

```diff
@@ -232,6 +234,7 @@ def self.acts_as_paranoid(options={})
       alias_method :destroy_without_paranoia, :destroy
 
       include Paranoia
+      include ParanoiaHelper
```

Comment:
When I read this line here I thought it was adding instance methods, then I discovered it was doing something to add class methods. It's a simple change, uses way less code, and is more idiomatic over a concern to include class level methods without any macros.